expression/integration_test.go

```diff
@@ -4398,3 +4398,8 @@ func (s *testIntegrationSuite) TestIssue10181(c *C) {
 	tk.MustExec(`insert into t values(9223372036854775807), (18446744073709551615)`)
 	tk.MustQuery(`select * from t where a > 9223372036854775807-0.5 order by a`).Check(testkit.Rows(`9223372036854775807`, `18446744073709551615`))
 }
+
+func (s *testIntegrationSuite) TestExprPushdownBlacklist(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustQuery(`select * from mysql.expr_pushdown_blacklist`).Check(testkit.Rows())
```

Comment:
How about adding more cases to test if `PbConverter.canFuncBePushed` works when `expr_pushdown_blacklist` is not empty?

Reply:
https://github.com/pingcap/tidb/pull/10688/files#diff-291f1a61cec176beffd747b77aff41d3R21

Do you mean this?

Reply:
Yes, my mistake.